projects/components/src/quick-search/quick-search.component.ts

```diff
@@ -561,12 +561,8 @@ export class QuickSearchComponent {
         return parentSection.subSections.some((section) => section.shouldShowText);
     }
 
-    showNoResults(searchSection: SearchSection): boolean {
-        // Show 'No Results' if the section is empty and the section title is shown
-        if (searchSection.result?.items?.length === 0 && searchSection.shouldShowText) {
-            return true;
-        }
-        return false;
+    hasNoResults(): boolean {
```

Comment:
Better have a single property that is calculated when all search providers finish.
Otherwise, the current approach, is calling `hasNoResults` from the template all the time.
Alternative approach is having:
changeDetection: ChangeDetectionStrategy.OnPush,